packages/next/src/lib/bundler.ts

```diff
@@ -0,0 +1,88 @@
+/// Utilties for configuring the bundler to use.
+export enum Bundler {
+  Turbopack,
+  Webpack,
+  Rspack,
+}
+/**
+ * Parse the bundler arguments and potentially sets the `TURBOPACK` environment variable.
+ *
+ * <p>NOTE: rspack is configured via next config which is chaotic so it is possible for this to be overridden later.
```

Comment:
```suggestion
 * NOTE: rspack is configured via next config which is chaotic so it is possible for this to be overridden later.
```

Is anything actually parsing these jsdoc-style comments aside from the LSP? I don't think we use HTML tags like this elsewhere?

Reply:
yeah im not sure.  i think i learned this pattern from javadoc lol